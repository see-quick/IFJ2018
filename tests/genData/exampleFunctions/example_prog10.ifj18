# Program to Count Number of Digits in an Integer

a = inputi
sum = 0

while a != 0 do
    a = a / 10
    sum = sum + 1
end

print "\nNumber of digits is: ", sum, "\n"
