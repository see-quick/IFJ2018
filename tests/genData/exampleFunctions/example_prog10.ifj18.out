50235
Number of digits is: 5
